File: backend/omegatechapi/src/main/java/com/example/omegatechapi/model/ticket.java
package com.example.omegatechapi.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;
import java.time.LocalDate;

@Data
@Entity

public class ticket {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)

    public String Resposta;
    public String ID;
    public String Titulo;
    public String Cliente;
    public String Prioridade;
    public String Tempo;
    public String Status;
    public String Descricao;
    public LocalDate DataCriacao;
    public int UsuarioId;
}

File: client-desktop/model/Ticket.cs
using System.Text.Json.Serialization;

namespace model {
    public class Ticket {
        [JsonPropertyName("id")]
        public long Id { get; set; }

        [JsonPropertyName("titulo")]
        public string Titulo { get; set; }

        [JsonPropertyName("descricao")]
        public string Descricao { get; set; }

        [JsonPropertyName("dataCriacao")]
        public DateTime DataCriacao { get; set; }

        [JsonPropertyName("prioridade")]
        public Prioridade Prioridade { get; set; }

        [JsonPropertyName("status")]
        public Status Status { get; set; }

        [JsonPropertyName("resposta")]
        public string Resposta { get; set; }

        [JsonPropertyName("cliente")]
        public Usuario Cliente { get; set; }

        [JsonPropertyName("tecnicoAtribuido")]
        public Usuario TecnicoAtribuido { get; set; }
    }
}
